src/kata4.humanCatDogYears.js

```diff
@@ -1,5 +1,11 @@
 const humanCatDogYears = (number) => {
-
-}
+    if(number === 1) {
+      return [number, 15, 15]
+    } else if(number === 2) {
+      return [number, 24, 24]
+    } else {
+      return [number, ((15 + 9) + ((number - 2) * 4)), ((15 + 9) + ((number - 2) * 5))]
+    };
```

Comment:
No need for `if / else if / else` here as you return out of the function in each block:

```js
if(number === 1) {
  return [number, 15, 15]
}

if (number === 2) {
  return [number, 24, 24]
}
  
return [number, ((15 + 9) + ((number - 2) * 4)), ((15 + 9) + ((number - 2) * 5))]
```